examples/spring-boot-on-kubernetes-with-jib-example/src/test/java/io/dekorate/example/SpringBootOnKubernetesTest.java

```diff
@@ -36,7 +36,6 @@ public void shouldContainDeployment() {
     Deployment d = findFirst(list, Deployment.class).orElseThrow(() -> new IllegalStateException());
     assertNotNull(d);
     Container container = d.getSpec().getTemplate().getSpec().getContainers().get(0);
-    assertTrue(container.getImage().contains("docker.io"));
```

Comment:
This line was deleted on purpose? Thanks to this assertion this issue was spotted. cc @iocanel